rllib/algorithms/algorithm_config.py

```diff
@@ -1026,7 +1026,7 @@ def build_env_to_module_connector(self, env):
             # Batch all data.
             pipeline.append(BatchIndividualItems(multi_agent=self.is_multi_agent()))
             # Convert to Tensors.
-            pipeline.append(NumpyToTensor())
+            pipeline.append(NumpyToTensor(device=device))
```

Comment:
That one was the missing part?!

Reply:
Yes, one of many, though. We do this on the LearnerConnector pipeline already and have to do this here too now.